src/libtiled/imagelayer.cpp

```diff
@@ -37,7 +37,9 @@
 using namespace Tiled;
 
 ImageLayer::ImageLayer(const QString &name, int x, int y):
-    Layer(ImageLayerType, name, x, y)
+    Layer(ImageLayerType, name, x, y),
+    mRepeatHorizontally(false),
+    mRepeatVertically(false)
```

Comment:
Please place the initialization at the declaration.

Reply:
Done.